.gitignore

```diff
@@ -33,3 +33,4 @@ build/local-config.mk
 *.pyo
 *.bak
 .*.swp
+/.vs
```

Comment:
What's this, and why is it hidden inside this commit?

Reply:
Visual Studio Files are ignored

Reply:
Why?
And (repeating myself): why is it hidden inside this commit?

Reply:
Hallo Max, this was not on purpose and you do not have to take this change into the main branch. Do you want me to change it, or do you want to make the change yourself.

Reply:
Amend the commit and remove it. This doesn't belong here.